zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -543,6 +545,16 @@ def get_postgresql_cluster_members(kube_client, cluster_id, alias, region, infra
         pod_namespace = obj['metadata']['namespace']
         service_dns_name = '{}.{}.svc.cluster.local'.format(labels['version'], pod_namespace)
 
+        # unfortunately, there appears to be no way of filtering these on the server side :(
+        pvc_name = obj['spec']['volumes'][0]['persistentVolumeClaim']['claimName']  # assume only one PVC
```

Comment:
I suggest wrapping in ``try..except`` in order not to fail the entire thing if some pvc/pv is not as expected.